README.md

```diff
@@ -70,6 +70,42 @@ new RollbarConfig
 </dt>
 <dd>A string URI which will be used as the WebClient proxy by passing to the WebProxy constructor.
 </dd>
+<dt>CheckIgnore
+</dt>
+<dd>Function called before sending payload to Rollbar. Return `true` to stop the error from being sent to Rollbar.
+</dd>
+<dt>Truncate
+</dt>
+<dd>Truncates the payload before sending it to Rollbar.
```

Comment:
a code example here as well please!

Reply:
any user-defined  .NET function delegate implementing following signature 
bool FunctionName(Payload payload)
can be assigned to this property.

Reply:
similar description goes for Transform action property...